### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by Kotoki1337**
Some strings i can't find the original location so i keep the original text untranslated like `Top line`, `Bottom line`.
At least it's readable now, should be a great translation.

### CODE REVIEW

**Comment by Kataiser** [comment]
Awesome, thank you! Will review soon

File: TF2 Rich Presence/localization.json [hunk 1]
```diff
@@ -444,7 +444,7 @@
         "Polish": "Obecne ustawienia są już domyślne.",
         "Russian": "Текущие настройки уже по умолчанию.",
         "Korean": "현재 설정은 이미 기본값입니다.",
-        "Chinese": "当前的设置已经是默认的。",
+        "Chinese": "当前的设置已是默认值。",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:447]:
```suggestion
        "Chinese": "当前设置已是默认值。",
```

File: TF2 Rich Presence/localization.json [hunk 2]
```diff
@@ -668,7 +668,7 @@
         "Polish": "Hej, wygląda na to, że Kataiser, twórca TF2 Rich Presence, jest w Twojej grze!\nPrzywitaj się ze mną, jeśli chcesz :)",
         "Russian": "Эй, похоже, что Kataiser, разработчик TF2 Rich Presence, в вашей игре!\nПоздоровайтесь со мной, если хотите:)",
         "Korean": "이봐,Tf2Rich Presence 의 개발자 인 Kataiser 가 게임에있는 것 같습니다!\n네가 원한다면 나에게 인사 해 :)",
-        "Chinese": "嘿，看来TF2丰富的存在感的开发者Kataiser在你的游戏里了!\n如果你愿意的话，请跟我打个招呼:)",
+        "Chinese": "嘿，TF2 Rich Presence 的开发者 Kataiser 加入了你现在的游戏！\n如果你愿意的话，可以跟我打个招呼 :)",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:671]:
```suggestion
        "Chinese": "嘿，TF2 Rich Presence 的开发者 Kataiser 正在你现在的服务器中！\n如果你愿意的话，快和他打个招呼吧 :)",
```

File: TF2 Rich Presence/localization.json [hunk 3]
```diff
@@ -682,7 +682,7 @@
         "Polish": "Ukryć typ gry (Casual, Konkurencyjny, MvM) ustawiony w kolejce do",
         "Russian": "Скрыть тип игры (Кэжуал, Комп, MvM) для которй вы в очереди",
         "Korean": "숨기기 게임 유형(캐주얼하고,경쟁적이고,데이터를 쉽게 마이닝하고 분석)대기",
-        "Chinese": "隐藏游戏类型（休闲，竞争，MvM）排队等候",
+        "Chinese": "隐藏排队时的游戏模式（休闲，竞争，MvM）",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:685]:
```suggestion
        "Chinese": "隐藏正在排队时的游戏模式（休闲，竞技，MvM）",
```

File: TF2 Rich Presence/localization.json [hunk 4]
```diff
@@ -724,7 +724,7 @@
         "Polish": "Timeout połączenia internetowego (dla aktualizacji i zapytań do serwera): ",
         "Russian": "Тайм-аут подключения к Интернету (для обновления и запроса к серверу): ",
         "Korean": "인터넷 연결 시간 초과(업데이트 및 서버 쿼리): ",
-        "Chinese": "互联网连接超时（用于更新程序和服务器查询）。",
+        "Chinese": "互联网连接超时（用于更新和服务器查询）。",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:727]:
```suggestion
        "Chinese": "网络连接超时（用于更新和服务器查询）：",
```

File: TF2 Rich Presence/localization.json [hunk 5]
```diff
@@ -1116,7 +1116,7 @@
         "Polish": "Licencja otwarta",
         "Russian": "Открытая лицензия",
         "Korean": "오픈 라이센스",
-        "Chinese": "开放许可证",
+        "Chinese": "查看许可证",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:1119]:
```suggestion
        "Chinese": "查看开源许可",
```

File: TF2 Rich Presence/localization.json [hunk 6]
```diff
@@ -1326,7 +1326,7 @@
         "Polish": "Stoi w kolejce do meczu na przyjęcie",
         "Russian": "В очереди на матч лобби",
         "Korean": "파티 경기 대기 중",
-        "Chinese": "排队等候派对的比赛",
+        "Chinese": "排队进入组队的比赛",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:1329]:
```suggestion
        "Chinese": "排队进入 Party 的比赛",
```

File: TF2 Rich Presence/localization.json [hunk 7]
```diff
@@ -1690,7 +1690,7 @@
         "Polish": "Ta wersja ({0}) jest nieaktualna (najnowsza wersja to {1}).",
         "Russian": "Эта версия ({0}) устарела (новейшая версия {1}).",
         "Korean": "이 버전({0})가 최신이(새로운 버전은{1}).",
-        "Chinese": "此版本（{0}）已过期（最新版本为{1}）。",
+        "Chinese": "当前版本（{0}）已过期（最新版本为 {1}）。",
```
Comment (Kotoki1337) [inline TF2 Rich Presence/localization.json:1693]:
```suggestion
        "Chinese": "当前版本（{0}）已过时（最新版本为 {1}）。",
```

**Review by Kotoki1337 — COMMENTED** [review]
Fix some weird translation

**Pull request merged**